src/Integration.UnitTests/LocalServices/ProjectCapabilityTestProjectIndicatorTests.cs

```diff
@@ -43,7 +43,10 @@ public void TestInitialize()
         {
             var serviceProvider = new ConfigurableServiceProvider(assertOnUnexpectedServiceRequest: true);
             var configurableVsProjectSystemHelper = new ConfigurableVsProjectSystemHelper(serviceProvider);
-            serviceProvider.RegisterService(typeof(IProjectSystemHelper), configurableVsProjectSystemHelper, true);
+
+            var mefHost = ConfigurableComponentModel.CreateWithExports(
+                MefTestHelpers.CreateExport<IProjectSystemHelper>(configurableVsProjectSystemHelper));
+            serviceProvider.RegisterService(typeof(SComponentModel), mefHost);
```

Comment:
This test didn't previously use any MEF components so I had a MEF host and set up the service provider to return it.